agent/bench-scripts/pbench-fio

```diff
@@ -957,3 +990,10 @@ if [[ "$postprocess_only" == "n" ]]; then
 	fi
 	rmdir $benchmark_run_dir/.running
 fi
+
+if [[ ${_frb_sts} -ne 0 || ${_gbs_sts} -ne 0 ]]; then
+	_exit_sts=1
+else
+	_exit_sts=0
+fi
+exit ${_exit_sts}
```

Comment:
Alternatively:
```
[[ ${_frb_sts} -eq 0 && ${_gbs_sts} -eq 0 ]]
exit
```
(and, you can omit the `exit`....)

Reply:
Wow that is subtle and tricky.  I'll do it, but I have added a comment since it is so sly.

Reply:
LOL

`[[` is a builtin shelll _command_!  😁 It does what it says on the tin....  (But, commenting things which are "surprising" is good.)

Reply:
OK, so it's not so much the presence of the `[[` "command" as it is the absence of the explicit `exit` command...so, yeah, a comment there is instructive.